src/mosdepth_genome_coverage_estimators.rs

```diff
@@ -222,6 +354,15 @@ impl PileupCountsGenomeCoverageEstimator {
             min_fraction_covered_bases: min_fraction_covered_bases
         }
     }
+    pub fn add_to_header(header_types: &mut HeaderTypes) -> &mut HeaderTypes{
+            let coverage_type = "Pileup Counts".to_string();
+            let index = "Index".to_string();
+            let st_dev = "Standard Deviation".to_string();
```

Comment:
I'm a little confused. This histogram struct outputs columns for stoit, genome, coverage and number of bases with that coverage (in that genome in that stoit). So, wouldn't the stdev of each row be 0? Guess I'm missing something?